server/http.go

```diff
@@ -37,28 +37,42 @@ func (p *Plugin) handleHTTPRequest(w http.ResponseWriter, r *http.Request) (int,
 	case "/uninstalled":
 		return p.handleHTTPUninstalled(w, r)
 
-	// OAuth end-points
-	case "/oauth/connect":
-		return p.handleHTTPOAuth2Connect(w, r)
-	case "/oauth/complete":
-		return p.handleHTTPOAuth2Complete(w, r)
-
-	case "/webhook",
-		"/issue_event":
-		return p.handleHTTPWebhook(w, r)
-
 	// User mapping page
 	case "/user-connect":
 		return p.handleHTTPUserConnect(w, r)
 	case "/user-disconnect":
 		return p.handleHTTPUserDisconnect(w, r)
+
+	// TODO use a router, make current JIRA instance register its own end-points
+
+	// Atlassian-connect based user mapping, used for JIRA Cloud
 	case "/user-config":
 		return p.handleHTTPUserConfig(w, r)
 	case "/user-config-submit":
 		return p.handleHTTPUserConfigSubmit(w, r)
 	case "/api/v1/userinfo":
 		return p.handleHTTPGetUserInfo(w, r)
 
+	// OAuth1 based user mapping, used in JIRA Server
+	case "/oauth1/public-key":
+		return p.handleHTTPOAuth1PublicKey(w, r)
+	case "/oauth1/connect":
+		return p.handleHTTPOAuth1Connect(w, r)
+	case "/oauth1/complete":
+		return p.handleHTTPOAuth1Complete(w, r)
+
+	// OAuth2 end-points - NOT FUNCTIONAL. They work with JIRA cloud, but there is
```

Comment:
Do we need them?

Reply:
We may use them for other Atlassian Cloud products, if they have App Links, like bitbucket. Not for JIRA, no.